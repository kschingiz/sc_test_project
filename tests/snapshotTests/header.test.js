import React from "react";
import Header from "../../components/header";
import renderer from "react-test-renderer";

it("Header renders correctly", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
