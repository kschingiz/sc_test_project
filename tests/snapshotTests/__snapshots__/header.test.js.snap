// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
Array [
  <div
    className="makeStyles-headerWrapper-1"
  >
    <div
      className="makeStyles-logoTitle-2"
    >
      Skipcard
    </div>
    <div
      className="makeStyles-link-3"
    >
      Create an event
    </div>
  </div>,
  <div
    className="makeStyles-headerBufferZone-4"
  />,
]
`;
